spirv_reflect.c

```diff
@@ -521,7 +593,37 @@ static SpvReflectPrvNode* FindNode(
   return p_node;
 }
 
-static SpvReflectTypeDescription* FindType(SpvReflectShaderModule* p_module, uint32_t type_id)
+static SpvReflectPrvEvaluationNode* FindEvaluationNode(
+  SpvReflectEvaluation* p_eval,
+  uint32_t             result_id)
+{
+  SpvReflectPrvEvaluationNode* p_node = NULL;
+  for (size_t i = 0; i < p_eval->node_count; ++i) {
+    SpvReflectPrvEvaluationNode* p_elem = &(p_eval->nodes[i]);
+    if (p_elem->result_id == result_id) {
+      p_node = p_elem;
+      break;
+    }
+  }
+  return p_node;
+}
+
+static SpvReflectPrvEvaluationNode* FindSpecIdNode(
+  SpvReflectEvaluation* p_eval,
+  uint32_t             specid)
+{
+  SpvReflectPrvEvaluationNode* p_node = NULL;
+  for (size_t i = 0; i < p_eval->node_count; ++i) {
+    SpvReflectPrvEvaluationNode* p_elem = &(p_eval->nodes[i]);
+    if (p_elem->specId == specid) {
+      p_node = p_elem;
+      break;
+    }
+  }
+  return p_node;
+}
+
+static SpvReflectTypeDescription* FindType(const SpvReflectShaderModule* p_module, uint32_t type_id)
```

Comment:
Module can also be const. Since this may be called in evaluation (when module is initialized and const), adding const here is needed.